artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPSessionCallback.java

```diff
@@ -233,6 +239,11 @@ public Object createSender(ProtonServerSenderContext protonSender,
       return consumer;
    }
 
+   private int getPriority(Map<Symbol, Object> properties) {
+      Integer value = properties == null ? null : (Integer) properties.get(PRIORITY);
```

Comment:
I would use a cast to `Number` and call `Number::byteValue`

Reply:
Users in AMQP land will expect Integer.

Qpid -https://qpid.apache.org/releases/qpid-broker-j-7.0.6/book/Java-Broker-Runtime-Consumers.html#Java-Broker-Runtime-Consumers-Prioirty